test/integration/contractor_removes_developers_from_project_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'test_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.